src/components/graphs-panel.tsx

```diff
@@ -21,6 +21,8 @@ interface IGraphsPanelProps {
   maxHeight: number;
   width: number;
   singleReads?: boolean;
+  sensorUnit?: string|null;
```

Comment:
Side note: Optional params in types use a tailing `?`. So these two statements are the same:
`usePrediction?: boolean;` and `usePrediction: boolean|undefined;`

(as an aside to my aside: unfortunately I have been inconsistent about using `undefined` or `null` for unset parameters. I don't know if there is a best practice or not, but based on this [article](https://blog.devgenius.io/typescript-when-to-use-null-undefined-or-empty-array-d45244ffc565) I will probably favor `undefined` in the future.)